Library/Formula/teensy_loader_cli.rb

```diff
@@ -12,15 +11,34 @@ class TeensyLoaderCli < Formula
     sha256 "dcd10140babb4d2937ce376c89e9c24a2e8046d2cabdad2cfdbc2542afa14471" => :mavericks
   end
 
+  devel do
+    url "https://github.com/PaulStoffregen/teensy_loader_cli.git", :revision => "0cca2087afb54173ce03109cabb1e29658703fc8"
+    version "2.1.devel"
+  end
+
+  option "with-libusb-compat", "Uses libusb instead od OS X HID api to connect with teensy boards. Available only for --devel and upwards."
+  depends_on "libusb-compat"  => :optional
+
+  def pour_bottle?
```

Comment:
Can kill this block. We handle this sort of logic automatically, and don't bottle `devel` at the moment regardless.